geode-core/src/main/java/org/apache/geode/management/internal/api/GeodeClusterManagementServiceFactory.java

```diff
@@ -157,7 +167,22 @@ private ClusterManagementService getClusterManagementServiceOnClient(String user
           "Unable to discover a locator that has ClusterManagementService running.");
     }
 
-    return create(cmsInfo.getHostName(), cmsInfo.getHttpPort(), null, new NoopHostnameVerifier(),
+    // if user didn't pass in a username and the locator requires credentials, use the credentials
+    // user used to create the client cache
+    if (cmsInfo.isSecured() && username == null) {
+      Properties securityProps = config.getSecurityProps();
+      username = securityProps.getProperty(AuthInitialize.SECURITY_USERNAME);
```

Comment:
At this point, if `username` is still null, do you think we should throw an exception saying that `security-username` is a required property?